avocado/core/job.py

```diff
@@ -366,7 +366,8 @@ def _log_avocado_datadir(self):
         job_log.info('')
 
     def _log_variants(self, variants):
-        for line in variants.to_str(1, 1, use_utf8=False).splitlines():
+        lines = variants.to_str(summary=1, variants=1)
```

Comment:
sure, I'll just have to double check why I removed the `use_utf8` which should be probably there...